source/help/getting-started/searching.rst

```diff
@@ -0,0 +1,116 @@
+Searching
+=========
+
+--------------
+
+The search box in Mattermost returns search results from any channel of
+which you’re a member.
+
+-  Searching multiple words returns results with any one of the words
+   listed.
+-  When results appear, click the **Jump** link to view that post in the
+   channel archive.
+-  You can expand the search results window by clicking the “Expand”
+   button in the top right corner.
+-  Use search modifiers, like ``from:dave``, to return results only from
+   certain people or in certain channels (see below)
+
+Like many search engines, highly common words like ``the``, ``which``,
+``are`` (known as “stop words”), as well as two-letter and one-letter
+search terms, are not shown in search because they typically return too
+may results.
```

Comment:
Propose we link the stop words for MySQL and Postgres databases here as well as below.

```
See database documentation on `MySQL`_ and `Postgres`_ for a full list of stop words.
```